sdk/v2/authx/api_client.go

```diff
@@ -5,6 +5,8 @@ import "github.com/brigadecore/brigade/sdk/v2/restmachinery"
 // APIClient is the root of a tree of more specialized API clients within the
 // authx package.
 type APIClient interface {
+	// RoleAssignments returns a specialized client for managing RoleAssignments.
+	RoleAssignments() RoleAssignmentsClient
```

Comment:
This is for managing system-level role assignments-- e.g. for roles like `ADMIN` or `PROJECT_CREATOR`.